mlir/tools/mlir-miopen-lib/mlir-miopen-lib.cpp

```diff
@@ -185,7 +241,7 @@ extern "C" void MlirLowerBin(MlirHandle mlirHandle) {
   PassManager pm(module.getContext(), PassManager::Nesting::Implicit);
 
   std::string triple = "amdgcn-amd-amdhsa";
-  BackendUtils utils(triple, s_arch, "");
+  BackendUtils utils(triple, handle->arch, "");
```

Comment:
Ok, so you use the passed in architecture instead of existing system runtime architecture. This make sense.

Does this arch field need to be stored as an individual member? The conv2d op carry this information.

Reply:
I had to add the s_arch as part of the rebase PR.. Just cleaning it up. It seems easier to cache it on the handle than look it up on the op.